packages/web-components/fast-components/src/anchor/anchor.definition.ts

```diff
@@ -38,49 +39,56 @@ export const fastAnchorDefinition: WebComponentDefinition = {
                 },
                 {
                     name: "download",
-                    description: "The download attribute",
+                    title: "Download",
+                    description: "The HTML download attribute of the anchor",
                     type: DataType.string,
                     default: undefined,
                     required: false,
                 },
                 {
                     name: "href",
-                    description: "The href attribute",
+                    title: "Href",
+                    description: "The HTML href attribute of the anchor",
                     type: DataType.string,
                     default: undefined,
                     required: false,
                 },
                 {
                     name: "hreflang",
-                    description: "The hreflang attribute",
+                    title: "Hreflang",
+                    description: "The HTML hreflang attribute of the anchor",
                     type: DataType.string,
                     default: undefined,
                     required: false,
                 },
                 {
                     name: "ping",
-                    description: "The ping attribute",
+                    title: "Ping",
+                    description: "The HTML ping attribute of the anchor",
                     type: DataType.string,
                     default: undefined,
                     required: false,
                 },
                 {
                     name: "referrerpolicy",
-                    description: "The referrerpolicy attribute",
+                    title: "Referrerpolicy",
+                    description: "The HTML referrerpolicy attribute of the anchor",
                     type: DataType.string,
                     default: undefined,
                     required: false,
```

Comment:
Looks like this was missing the actual values.

```suggestion
                    required: false,
                    values: [
                        "no-referrer",
                        "no-referrer-when-downgrade",
                        "origin",
                        "origin-when-cross-origin",
                        "same-origin",
                        "strict-origin",
                        "strict-origin-when-cross-origin",
                        "unsafe-url"
                    ],
```